tests/test_config_loader.py

```diff
@@ -157,15 +176,16 @@ def test_load_changing_group_in_default(
             ),
         ],
     )
-    def test_load_changing_group_and_package_in_default(
+    def test_load_changing_group_and_package_in_default_deprecated(
```

Comment:
It's very hard to review your diff for some reason. can you not rename the functions? somehow this is confusing github.